src/backend/distributed/utils/foreign_key_relationship.c

```diff
@@ -259,20 +274,10 @@ static void
 GetConnectedListHelper(ForeignConstraintRelationshipNode *node, List **adjacentNodeList,
 					   bool isReferencing)
```

Comment:
One day we should convert this boolean to an enum and do the same for the helper function (`GetNeighbourList`) that I implemented in this pr